python/scripts/build-wheels/Dockerfile

```diff
@@ -3,7 +3,18 @@ FROM quay.io/pypa/manylinux2010_x86_64
 RUN yum install git -y
 RUN git config --global --add safe.directory /code
 
-RUN curl https://sh.rustup.rs -sSf | sh -s -- -y --profile minimal --default-toolchain 1.63
+# update glibc to 2.17 for CentOS 6
+RUN curl -OJ http://copr-be.cloud.fedoraproject.org/results/mosquito/myrepo-el6/epel-6-x86_64/glibc-2.17-55.fc20/glibc-2.17-55.el6.x86_64.rpm
+RUN curl -OJ http://copr-be.cloud.fedoraproject.org/results/mosquito/myrepo-el6/epel-6-x86_64/glibc-2.17-55.fc20/glibc-common-2.17-55.el6.x86_64.rpm
+RUN curl -OJ http://copr-be.cloud.fedoraproject.org/results/mosquito/myrepo-el6/epel-6-x86_64/glibc-2.17-55.fc20/glibc-devel-2.17-55.el6.x86_64.rpm
+RUN curl -OJ http://copr-be.cloud.fedoraproject.org/results/mosquito/myrepo-el6/epel-6-x86_64/glibc-2.17-55.fc20/glibc-headers-2.17-55.el6.x86_64.rpm
+
+RUN rpm -Uvh glibc-2.17-55.el6.x86_64.rpm
+RUN glibc-common-2.17-55.el6.x86_64.rpm
+RUN glibc-devel-2.17-55.el6.x86_64.rpm
+RUN glibc-headers-2.17-55.el6.x86_64.rpm
```

Comment:
Doesn't this make us no longer compatible with the manylinux wheels? The absence of a modern glibc is why we where using 1.63 toolchain here before.